qc.ipynb

```diff
@@ -0,0 +1,619 @@
+      "text/plain": [
+       "<Figure size 1700x2300 with 18 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from matplotlib.colors import ListedColormap\n",
+    "\n",
+    "\n",
+    "# choose 18 random files\n",
+    "np.random.seed(907)\n",
+    "idx = np.random.randint(0, 50, 18)\n",
+    "all_fps = list(out_risk_dir.glob(\"*.tif\"))\n",
+    "fps = [all_fps[i] for i in idx]\n",
+    "cmap = ListedColormap([\"green\", \"orange\", \"red\"])\n",
+    "\n",
+    "fig, axs = plt.subplots(6, 3, figsize=(17, 23), sharex=True, sharey=True)\n",
+    "fig.patch.set_facecolor(\"white\")\n",
+    "for ax, fp in zip(axs.flatten(), fps):\n",
+    "    ax.get_xaxis().set_visible(False)\n",
+    "    ax.get_yaxis().set_visible(False)\n",
+    "    era, model, scenario, snow = fp.name.split(\"_\")[-4:]\n",
+    "    snow = snow.replace(\".tif\", \"\")\n",
+    "    with rio.open(fp) as src:\n",
+    "        show(src, interpolation=\"none\", cmap=cmap, ax=ax)\n",
+    "\n",
+    "        ax.set_title(f\"{era} {model} {scenario} {snow} snow\")\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf652e2d",
+   "metadata": {},
+   "source": [
+    "## Algorithm validation\n",
+    "\n",
+    "Given the complexity of the model / algorithm which results in the final risk class values, it would be wise to implement it from scratch for a single pixel as a sanity check the various datasets produced.\n",
+    "\n",
+    "Choose a single model anf scenario, and a set of four years. This is the minimum requried to get a single year's risk value because that depends on risk components from the two prior years, and the first of those two year's risk components will depend on the preceding winter which then requires the year before that as well."
```

Comment:
```suggestion
    "Choose a single model and scenario, and a set of four years. This is the minimum required to get a single year's risk value because that depends on risk components from the two prior years, and the first of those two year's risk components will depend on the preceding winter which then requires the year before that as well."
```